molo/profiles/views.py

```diff
@@ -21,14 +21,28 @@ def register(request):
                 username=form.cleaned_data['username'],
                 password=form.cleaned_data['password'],
             )
-            user.profile.date_of_birth = form.cleaned_data['date_of_birth']
             user.profile.save()
-            return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
+            return HttpResponseRedirect(form.cleaned_data.get(
+                                        'molo.profiles:registration_done'))
         return render(request, 'profiles/register.html', {'form': form})
     form = RegistrationForm()
     return render(request, 'profiles/register.html', {'form': form})
 
 
+@csrf_protect
+def register_done(request):
```

Comment:
This is the old way of doing Form handling. The newer way of doing this is using [Class based views](https://github.com/praekelt/molo.profiles/blob/feature/issue-9-create-registration-done-form-with-DOB/molo/profiles/views.py#L59-L71)